### PR TITLE
CTCP-4785: IE170 - Populating Consignment. Mode of Transport at the Border in IE170 user answers

### PR DESCRIPTION
**Pull request by joecefnhedog**
image

this is the CYA coming from transformed 15 data.

image

The form is also filled in.

### CODE REVIEW

File: app/utils/transformer/DepartureDataTransformer.scala [hunk 1]
```diff
@@ -43,7 +50,9 @@ class DepartureDataTransformer @Inject() (
       transportEquipmentTransformer.transform andThen
       containerIdTransformer.transform andThen
       sealTransformer.transform andThen
-      limitDateTransformer.transform
+      limitDateTransformer.transform andThen
+      modeOfTransportAtTheBorderTransformer.transform
+    addBorderModeOfTransportYesNoTransformer.transform
```
Comment (jackheslop96) [inline app/utils/transformer/DepartureDataTransformer.scala:55]: Are we perhaps missing a `andThen` here?

Comment (joecefnhedog) [reply app/utils/transformer/DepartureDataTransformer.scala]: Yeah, just amended that

**Review by jackheslop96 — DISMISSED** [review]

File: app/utils/transformer/DepartureDataTransformer.scala (hunk 1 above)
Comment (justega247) [reply app/utils/transformer/DepartureDataTransformer.scala]: @joecefnhedog can the clean-up logic on the AddBorderModeOfTransportYesNoPage be then updated?

Comment (joecefnhedog) [reply app/utils/transformer/DepartureDataTransformer.scala]: Good spot, I reckon we remove this functionality to ensure we have captured all the cases where 15 data may be getting removed.

Comment (justega247) [reply app/utils/transformer/DepartureDataTransformer.scala]: Once all the pages that are affected by the clean-up logic have started using the transform implementations then I believe the logic can be updated.

**Review by justega247 — APPROVED** [review]

**Pull request merged**